autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -127,6 +127,7 @@ def fit(train_data, label, tuning_data=None, time_limits=None, output_directory=
                     Optimizes result immediately for deployment by deleting unused models and removing training artifacts.
                     Often can reduce disk usage by ~2-4x with no negatives to model accuracy or inference speed.
                     This will disable numerous advanced functionality, but has no impact on inference.
+                    This will make certain functionality less informative, such as `predictor.leaderboard()` and `predictor.fit_summary()`.
```

Comment:
can we explain why, eg something like:

"Because unused models will be deleted under this preset, methods like `predictor.leaderboard()` and `predictor.fit_summary()` will no longer show the full set of models that were trained during `fit()`

Reply:
Added